charts/clusterapi-resources/values.yaml

```diff
@@ -140,6 +140,12 @@ machineDeployments:
   #     key: nvidia.com/gpu
   # String of comma separated key=value pairs to add to nodes created by this machineDeployments
   # node-labels: custom-label=hello-world
+  # The deployment strategy to use to replace existing machines with new ones
+  # strategy:
+  #   remdiation: object
```

Comment:
Typo?

Reply:
Thanks, ill fix when I have a real example to put in